src/components/common/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  Button,
+  ButtonGroup,
+  Icon,
+  UseRadioGroupProps,
+  useRadio,
+  useRadioGroup,
+} from "@chakra-ui/react"
+
+interface RadioButtonProps {
+  label?: string
+  icon?: React.ElementType
+  isChecked?: boolean
+  value: string
+}
+
+interface RadioButtonGroupProps {
+  radioGroupProps: UseRadioGroupProps
+  options: Array<{ value: string; label?: string; icon?: React.ElementType }>
```

Comment:
Could we use `RadioButtonProps` here? Just to remove the unnecessary duplication.